src/freenas/usr/local/lib/middlewared_truenas/plugins/enclosure_/map.py

```diff
@@ -19,6 +19,17 @@
             MappingSlot(0, 4, False),
         ]),
     ]),
+    ProductMapping(re.compile(r"FREENAS-MINI-3.0-X\+?"), [
```

Comment:
the X+ will probably have different mapping because its a different motherboard. Will need to wait to confirm though